File: dream_framework_0607/src/www/dream/com/framework/test/Reply.java
package www.dream.com.framework.test;

import www.dream.com.framework.lengPosAnalyzer.HashTarget;

public class Reply {
	private String content;
	
	@HashTarget
	private Party writer;

	@HashTarget // 내용물을 분석하여 여기 긴 문장에서 핵심 단어를 추출해낼것 
	public String getContent() {
		return content;
	}
	
	public Party getWriter() {
		return writer;
	}

	public void setContent(String content) {
		this.content = content;
	}

	public void setWriter(Party writer) {
		this.writer = writer;
	}
}

File: bb008_replyWithAjax/src/main/webapp/resources/js/post/reply.js
/**
 * 
 */
console.log("reply.js 연동 성공!!!");

/** 함수 정의 영역 */
var replyService = (function() {
	
	function add() {
		console.log("add 실행 됨!");
	}
	
	return {add:add};
})();